modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -130,7 +130,7 @@ def claimant?
     end
 
     def domestic_phone?
-      phone_country_code == '1'
+      phone_country_code.blank? || phone_country_code == '1'
```

Comment:
I'm not sure it's correct to change this here, but I noticed that this would consider phone numbers without an explicit country code to be international.

[This test case](https://github.com/department-of-veterans-affairs/vets-api/pull/11866/files#diff-6c00d0ac625760ebd3e350ea23f5fca4cf155991c5711f048d280ecfc736da5cR116) fails if this change isn't made - I could fix this in the FormData logic instead, but I thought I'd try here first. Happy to update if this change seems too risky.